### PR TITLE
Added webGroup class.

### PR DESCRIPTION
**Pull request by PatrykSzwer**
Added webGroup class, refactored WebTemplate and WebSection to use expresion-bodied properties.
This is needed for https://github.com/StarcounterApps/Website/issues/122.

### CODE REVIEW

**Comment by PatrykSzwer** [comment]
First failed build is from incorrect branch (develop) with SC version 2.4.  Second build is on master branch.

**Comment by un-tone** [comment]
The code LGTM. But I think we should do something with build failings (for example I don't have access to the TC, so I can't know what is exactly happening there with current branch). @warpech could you look at this?

**Comment by warpech** [comment]
Thanks for the info. I will add it here: https://github.com/Starcounter/AdminTrack/issues/299

File: Ring6/Web/WebGroup.cs [hunk 1]
```diff
@@ -0,0 +1,11 @@
+﻿using Simplified.Ring1;
+using Starcounter;
+
+namespace Simplified.Ring6
+{
+    [Database]
+    public class WebGroup : Something
```
Comment (warpech) [inline Ring6/Web/WebGroup.cs]: I think this class should be renamed `WebTemplateGroup`

Comment (PatrykSzwer) [inline Ring6/Web/WebGroup.cs]: Done!

**Pull request merged**